IMC_WebDev/MiniCityBlog.md

```diff
@@ -0,0 +1,96 @@
+
+
+Got it? Great!
+
+Next, we have to actually create our HTML tag. This just means that we are beginning to write our HTML code, but first we must actually tell our computer that we will be writing in HTML. To do this, simply type “html” on your second line. Your code should look like this so far: <img src=https://lh4.googleusercontent.com/xcemOKcTNUUlo5yAwQvAyAaMhLiwskacjc1LTDy0P0yoaSUpssBCjlURYTfSYsw-yHPBpXGgJCK_Y76lxqLPbjmiEx7WSMxJ5v-OclRHJo1J7YkQNaqI9sR-el_PM8qnaUp0rjvt>
+
+Now that we have established the fact that we will be writing in HTML, it’s time to get into the fun stuff. We start by writing “head” in our third line of code. This tells our editor that the information for our header will be stored here.
+
+After we write our “head” tag, we know that we should give our project a title. Let’s title our project “My Mini City”. This is how your code should look now: <img src=https://lh3.googleusercontent.com/1ZecOiCyLV4wjTHfQsGzQSuJJGePIZp46IQkE29VVEmbHz1WUeS5tsDMs0iiKPVIFjCyic2dk49RRXEbnbwYrJFjGgykmT4rVDqWedqytukmAUpdbSiDKXYdvogTciuwNopK4Vzl>
+
+You may notice that there is a /title tag after our title’s name. This is called a closing tag. Closing tags have a “/” before they state which tag they are closing. This just tells our editor that we are no longer editing this tag.
+
+To create our body section, simply type body on your next line.
+
+Now, let’s make our classes! Let’s think about what we want our city to look like.
+We know that we want the following objects in our city:
+The ground
+A skyscraper
+A smaller building
+A road
+A bridge
+An ocean
+A waterfall
+A car
+
+In order to eventually create these objects, we have to make classes for them. Classes are basically like blank notebooks that we use to establish the fact that we are going to create an object. Then, we actually create it in CSS!
+
+We create classes using the "div" tag. Like this: <img src=https://lh6.googleusercontent.com/1iPgi651LuYWiyo5AnInaCkxrpUMpnVAeBFdxIiEtEpILCPIm1JixLkgK5oUWdsnLYMyOxsO2LjNdGGBz47F-g-yJpUt2dtV4mnathNFa6TGKwOJ2KJoXVRXrY2b9M0W6eQ6r6nR>
+As you can see, we just created a class called “ground”. Now our editor knows that we are planning to create an object.
+
+We do the same thing with all of our other objects. When you are done, it should look something like this: <img src=https://lh4.googleusercontent.com/KvgGx3EetIK7wY0oXVeEFvrvaznJTFkrwE6fwTi0HoYN4Rymi5SmJj6WAc-g2BUgAiFMjVm3WNUOdvusnb-0-cbewzMQDYZhIWOdrJRLtGd-KC2Ox1Ps7MVcq8Ev4bG5NOUirUz3> <img src=https://lh4.googleusercontent.com/QPwNwwgKTGt_I6I4pmBxnd5dvlew7OocTCup8YiqBB8DTmmnyaWZ2Qi5TK_GBboP6gjdJgObXWbt1-pIjPF95GlkQSGlZcfw0Z3Wa7amH-24IBkAMO4Z6mXmepX5uNoRb5kh9xao>
+
+
+This looks like a huge chunk of code right now, but let’s break it down!
+
+For every object in our city, we create a div class for it. That way, we can edit it using our CSS. HTML is just our canvas for all of our objects, while CSS is like our paint!
```

Comment:
"HTML is just our canvas for all of our objects, while CSS is like our paint!". This is not technically accurate. That makes it seem like CSS does all of the work in creating a website. To use a similar analogy, it would be more like HTML is the black-and-white sketch in a notebook, while CSS is used to add color to that sketch. CSS is what makes it pretty, but there would be nothing to color without HTML!